stdpopsim/catalog/DroMel/__init__.py

```diff
@@ -44,20 +32,18 @@
 )
 
 _chromosomes = []
-for line in _chromosome_data.splitlines():
-    name, length = line.split()[:2]
+for name, data in genome_data.data["chromosomes"].items():
     _chromosomes.append(stdpopsim.Chromosome(
-        id=name, length=int(length),
+        id=name, length=int(data["length"]),
+        synonyms=data["synonyms"],
         mutation_rate=5.49e-9,  # citation: _SchriderEtAl
         recombination_rate=8.4e-9))  # WRONG, underestimate used in S&S!
```

Comment:
I think this introduces a regression---the mito and Y will have a non-zero recombination rate. See #405.

Reply:
Well spotted, ta.

Reply:
Fixed.